cmd/occollector/app/builder/sampling_builder.go

```diff
@@ -67,27 +67,27 @@ type PolicyCfg struct {
 	Configuration interface{}
 }
 
-// NumericTagFilterCfg holds the configurable settings to create a numeric tag filter
+// NumericAttributeFilterCfg holds the configurable settings to create a numeric attribute filter
 // sampling policy evaluator.
-type NumericTagFilterCfg struct {
+type NumericAttributeFilterCfg struct {
 	// Tag that the filter is going to be matching against.
-	Tag string `mapstructure:"tag"`
-	// MinValue is the minimum value of the tag to be considered a match.
+	Attribute string `mapstructure:"attribute"`
+	// MinValue is the minimum value of the attribute to be considered a match.
 	MinValue int64 `mapstructure:"min-value"`
-	// MaxValue is the maximum value of the tag to be considered a match.
+	// MaxValue is the maximum value of the attribute to be considered a match.
 	MaxValue int64 `mapstructure:"max-value"`
 }
 
-// StringTagFilterCfg holds the configurable settings to create a string tag filter
+// StringAttributeFilterCfg holds the configurable settings to create a string attribute filter
 // sampling policy evaluator.
-type StringTagFilterCfg struct {
+type StringAttributeFilterCfg struct {
 	// Tag that the filter is going to be matching against.
-	Tag string `mapstructure:"tag"`
-	// Values is the set of tag values that if any is equal to the actual tag valueto be considered a match.
+	Attribute string `mapstructure:"attribute"`
```

Comment:
since it already has attribute on the name of the filter it will be nice to have this renamed to `key`